packages/likes/src/likes.tsx

```diff
@@ -46,60 +53,67 @@ function Likes(props: Props) {
       return;
     }
 
-    if (!currentUser.role || !hasRole(currentUser, props.config.votes.requiredUserRole)) {
+    if (
+      !currentUser.role ||
+      !hasRole(currentUser, props.config.votes.requiredUserRole)
+    ) {
       // login
       session.set('osc-idea-vote-pending', { [idea.id]: value });
-      return document.location.href = props.config.login.url;
+      return (document.location.href = props.config.login.url);
     }
 
     let change = {};
     if (idea.userVote) change[idea.userVote.opinion] = -1;
 
     await idea.submitLike({
-      opinion: value
-    })
+      opinion: value,
+    });
 
     setIsBusy(false);
-
   }
 
   return (
-    <>
-      <div id="like-widget-container" onClick={e => doVote(e, 'yes')}>
-        <h3 className="like-widget-title">Likes</h3>
-        <div className="like-option">
-          <section className="like-kind">
-            <i className="ri-thumb-up-line"></i>
-            <div>Voor</div>
-          </section>
-
-          <section className="like-counter">
-            <p>
-              {idea.yes < 10 ? idea.yes.toString().padStart(2, '0') : idea.yes}
-            </p>
-          </section>
-        </div>
-        <div className="like-option" onClick={e => doVote(e, 'no')}>
-          <section className="like-kind">
-            <i className="ri-thumb-down-line"></i>
-            <div>Tegen</div>
-          </section>
-
-          <section className="like-counter">
-            <p>
-              {idea.no < 10 ? idea.no.toString().padStart(2, '0') : idea.no}
-            </p>
-          </section>
-        </div>
-
-        <div className="progressbar-container">
-          <ProgressBar progress={(idea.yes / necessaryVotes) * 100} />
-          <p className="progressbar-counter">
-            {idea.yes} /{necessaryVotes}
+    <div
+      className="osc"
+      id="like-widget-container"
```

Comment:
Is het theoretisch mogelijk dat je deze like widget meerdere keren op de pagina hebt? bijvoorbeeld in een lijst met resources dat je snel een like kunt achterlaten? En wordt dit `id` ergens voor gebruikt?

Reply:
Deze widget kan je wel op die manier gebruiken inderdaad. Wel moeten nog denk ik de verkleinde varianten ondersteund kunnen worden. Nu is alleen nog de meest volledige variant te gebruiken. Ik maak hier wel een ticket apart voor om de andere stijlen te kunnen gebruiken.

<img width="1296" alt="Screenshot 2023-11-22 at 13 33 01" src="https://github.com/openstad/openstad-headless/assets/5735209/47ac0544-181e-49a3-9b45-5ff77adb9401">